doc/modules/ROOT/pages/projects.adoc

```diff
@@ -774,3 +774,6 @@ external command or an Emacs Lisp function:
 ----
 (setq projectile-test-cmd #'custom-test-function)
 ----
+
+In addition caching of commands can be disabled by setting the variable
```

Comment:
Might be good to mention that CMake projects are most likely to benefit from this.

Reply:
Added a line.